desc/objectives/objective_funs.py

```diff
@@ -938,6 +942,24 @@ def build(self, eq=None, use_jit=True, verbose=1):
         self._check_dimensions()
         self._set_dimensions(eq)
         self._set_derivatives()
+
+        # set quadrature weights if they haven't been
+        if hasattr(self, "_constants") and ("quad_weights" not in self._constants):
+            if self._coordinates == "":
+                w = jnp.ones((self.dim_f,))
+            elif self._coordinates == "rtz":
+                w = self._constants["transforms"]["grid"].weights
+                w *= jnp.sqrt(self._constants["transforms"]["grid"].num_nodes)
```

Comment:
For my own understanding, is the logic behind this method of making independent of resolution as follows

We have some error vector $f_i$ with $i=0->N$ where $N$ is num_nodes.

when we scale, we multiply by `grid.weights`=$d\rho d\theta d\zeta$ and then by $\sqrt{N}$

`grid.weights` is $\approx (1/N^{1/3})(2\pi/{N^{1/3})(2\pi/{N^{1/3}) \approx 4\pi^2 / N$ if we assume that the grid points are linearly spaced and that the number of unique rho, theta, and zeta nodes are the same, so $N_{rho} = N^{1/3}$ etc. 

So when we square each scaled error and sum, we get something like $\sum_i f_i^2 * 16 \pi^4/N^2 N = 16\pi^4 \sum_i f_i^2 / N \approx 16 \pi^2 f_i^2/N * N \approx 16 \pi^2 f_i^2$ where I assumed $f_i$ is constant across grid points, and this is thus independent of grid resolution